src/main/java/seedu/bigpp/command/buildercommand/BuilderListComponentCommand.java

```diff
@@ -52,15 +52,15 @@ public String executeCommand(DataStorage dataStorage) throws BuilderMissingListE
         if (userInputString.equals("")) {
             throw new BuilderMissingComponentException();
         }
-        if (!dataStorage.stringToComponentListMap.containsKey(componentType)) {
+
+        if (!dataStorage.getStringToComponentListMap().containsKey(componentType)) {
             throw new BuilderIncorrectComponentException();
         }
 
-        ComponentList<?> componentList = dataStorage.stringToComponentListMap.get(componentType);
+        ComponentList<?> componentList = dataStorage.getStringToComponentListMap().get(componentType);
```

Comment:
is this the only place where the change for `getStringToComponentListMap()` is needed?

Reply:
yes i think so